percy-mirror-verifier/src/percy.ts

```diff
@@ -21,17 +21,13 @@ export type PercySnapshot = {
   id: string;
   attributes: {
     name: string;
+    fingerprint: string | null;
     'review-state':
       | 'approved'
       | 'unreviewed'
       | 'changes_requested'
       | 'failed'
       | null;
-    'review-state-reason':
-      | 'no_diffs'
-      | 'auto_approved_branch'
-      | 'user_approved'
-      | null;
```

Comment:
Does this attribute not exist or are you just not using it anymore? I generally like to leave unused attributes in type definitions to help with future refactoring.

Reply:
It's just unused. The thing is, the type itself has dozens of fields that aren't used, and I didn't want to "pollute" the type system with the unused one. I can add a comment about how these types are incomplete if you think that makes sense?